llvm-bazel/.bazelrc

```diff
@@ -68,6 +68,23 @@ build:generic_gcc --copt=-Wno-misleading-indentation --host_copt=-Wno-misleading
 # Use `-Werror` for GCC to make sure warnings don't slip past.
 build:generic_gcc --copt=-Werror --host_copt=-Werror
 
+###############################################################################
+# Windows specific flags for building with GCC. Used on top of generic_gcc
+###############################################################################
+
+# Other windows toolchains have these by default
+build:mingw_gcc --cxxopt=-DHAVE_UNISTD_H=1 --host_cxxopt=-DHAVE_UNISTD_H=1
+build:mingw_gcc --linkopt=-lole32 --host_linkopt=-lole32
+build:mingw_gcc --linkopt=-luuid --host_linkopt=-luuid
+build:mingw_gcc --linkopt=-lversion --host_linkopt=-lversion
```

Comment:
I moved them into the specific libraries. I also tried compiling them with `msvc`, which would complain about LNK4044... I tried adding `(host_)linkopt=/IGNORE:4044` but no luck, the warnings are still there.

Reply:
So as currently configured (with them on the specific targets), this adds LNK4044 warnings to MSVC? That's worse than having them in the bazelrc, I think. I can take a look next week about trying to detect mingw properly, but otherwise let's just leave things in the bazelrc as you had previously. @chandlerc thoughts?

Reply:
When trying to run with 
```
bazelisk build --config=msvc --compiler=msvc @llvm-project//llvm:llvm-tblgen
...
LINK : warning LNK4044: unrecognized option '/lole32'; ignored
LINK : warning LNK4044: unrecognized option '/luuid'; ignored
...
```

I _think_ these [are safe](https://docs.microsoft.com/en-us/cpp/error-messages/tool-errors/linker-tools-warning-lnk4044?view=msvc-160), albeit annoyingly verbose?

Reply:
Yeah warning spam is IMO indirectly unsafe though because it makes people ignore other warnings. I think it's not fair to degrade the MSVC build experience to add support for mingw_gcc

Reply:
Done.